_maps/RandomZLevels/Academy.dmm

```diff
@@ -2958,7 +2958,8 @@
 /turf/open/floor/plating,
 /area/awaymission/academy/academyengine)
 "lL" = (
-/mob/living/simple_animal/slaughter{
+/mob/living/simple_animal/hostile/imp/slaughter{
+	faction = list("hell", "neutral");
```

Comment:
Could you make a variant instead of var-editing it?

Reply:
its a demon in an unaccessible part (behind unbreakable windows and walls) of a disabled away mission with 2 vars changed

Reply:
```suggestion
/mob/living/simple_animal/hostile/imp/slaughter/demon_in_an_unaccessible_part_behind_unbreakable_windows_and_walls_of_a_disabled_away_mission_with_2_vars_changed
```